packages/module/src/AssistantMessageEntry/AssistantMessageEntry.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Icon, Label, Split, SplitItem, TextContent, LabelProps } from '@patternfly/react-core';
+import { createUseStyles } from 'react-jss';
+import classnames from "clsx";
+
+import RobotIcon from '@patternfly/react-icons/dist/js/icons/robot-icon';
+
+const useStyles = createUseStyles({
+  chatbot: {
+    marginRight: "40px",
+  },
+  bubble: {
+    border: "1px solid var(--pf-v5-global--BackgroundColor--dark-400)",
+    borderRadius: "14px",
+    padding: "var(--pf-v5-global--spacer--sm) var(--pf-v5-global--spacer--md) var(--pf-v5-global--spacer--sm) var(--pf-v5-global--spacer--md)",
+    maxWidth: "100%",
+    wordWrap: "break-word",
+  },
+  label: {
+    // not working
+    "BorderColor": "var(--pf-v5-c-label--m-red__icon--Color)",
+  }
+})
+
+interface AssistantMessageEntryProps {
+  title: string;
+  options?: {
+    title: string;
+    props?: LabelProps
+  }[]
+}
+
+export const AssistantMessageEntry: React.FC<AssistantMessageEntryProps> = ({ title, options }) => {
+  const classes = useStyles();
+  return (
+    <div className="pf-v5-u-mb-md">
+      <Split className={classes.chatbot}>
+        <SplitItem>
+          <Icon size="lg" className="pf-v5-u-mr-sm pf-v5-u-pt-md">
+            <RobotIcon />
+          </Icon>
+        </SplitItem>
+        <SplitItem className={classnames(classes.bubble," pf-u-background-color-200")}>
+          <TextContent className="pf-v5-u-font-size-sm">
+            {title}
```

Comment:
Could this be a `prop.children` instead? It would give more flexibility, like using a markdown component or similar.